itest/deb_package_itest.sh

```diff
@@ -15,10 +28,11 @@ function run {
   pidtree-bcc $@
 }
 
+echo $@
```

Comment:
I guess this is here for debugging, which is actually fine for an itest imo, maybe just add some context, e.g. `echo "Packaging arguments: $@"`

Reply:
Fixed